SPECS/abseil-cpp/abseil-cpp.signatures.json

```
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "abseil-cpp-20220623.0.tar.gz": "4208129b49006089ba1d6710845a45e31c59b0ab6bff9e5788a87f55c5abd602"
+  "abseil-cpp-20230802.1.tar.gz": "987ce98f02eefbaf930d6e38ab16aa05737234d7afbab2d5c4ea7adbe50c28ed"
  }
 }
```

nit - end of file newline
```suggestion
}

```